BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBStudyParticipant.h

```diff
@@ -35,38 +35,41 @@
 
 #import "SBBStudyParticipantCustomAttributes.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @protocol _SBBStudyParticipant
 
 @end
 
 @interface _SBBStudyParticipant : SBBBridgeObject
 
-@property (nonatomic, strong) SBBStudyParticipantCustomAttributes* attributes;
+@property (nullable, nonatomic, strong) SBBStudyParticipantCustomAttributes* attributes;
 
-@property (nonatomic, strong) NSDate* createdOn;
+@property (nullable, nonatomic, strong) NSDate* createdOn;
 
-@property (nonatomic, strong) NSSet<NSString *>* dataGroups;
+@property (nullable, nonatomic, strong) NSSet<NSString *>* dataGroups;
 
-@property (nonatomic, strong) NSString* email;
+@property (nullable, nonatomic, strong) NSString* email;
 
-@property (nonatomic, strong) NSString* externalId;
+@property (nullable, nonatomic, strong) NSString* externalId;
 
-@property (nonatomic, strong) NSString* firstName;
+@property (nullable, nonatomic, strong) NSString* firstName;
 
-@property (nonatomic, strong) NSString* id;
+@property (nullable, nonatomic, strong) NSString* id;
```

Comment:
Should this be non-null.

Reply:
Again, `id` seems like something I would expect to be non null.

Reply:
This one is a weird case. Until you've signed up, the id hasn't been assigned by the server, but this is the object that gets filled in and submitted in the signUp call.